src/components/pages/donation_form/AddressForms.vue

```diff
@@ -176,7 +176,9 @@ const {
 	onFieldChange,
 	onAutofill,
 } = useAddressFunctions( { addressValidationPatterns: addressValidationPatterns.value }, store );
-const mailingList = useMailingListModel( store );
+// const mailingList = useMailingListModel( store );
```

Comment:
Please undo this change - we still need to use the store for the regular donation form, because the values form the `<SubmitForm>` component come from the store. If you use a local `ref`, the values in the store will never change and the user choice won't be submitted. Setting the default for the donation form happens in `createInitialDonationAddressValues`

Reply:
Done.